src/sankoff_handler.hpp

```diff
@@ -11,81 +11,108 @@
 #include "site_pattern.hpp"
 #include "node.hpp"
 #include "driver.hpp"
+#include "pv_handler.hpp"
 
 // All partial vectors for all nodes at one site
-using SankoffPartial = Eigen::Matrix<double, Eigen::Dynamic, 4>;
+using SankoffPartial = NucleotidePLV;
 // Each SankoffPartial represents a calculations for one site
 using SankoffPartialVec = std::vector<SankoffPartial>;
+// references for SankoffPartials
+using SankoffPartialRef = Eigen::Ref<SankoffPartial>;
+using SankoffPartialRefVec = std::vector<SankoffPartialRef>;
 
 class SankoffHandler {
  public:
-  static inline const size_t partial_type_count_ = 3;
-  enum class SankoffPartialType : size_t { PLeft = 0, PRight = 1, Q = 2 };
-  using SankoffTypeIterator =
-      EnumIterator<SankoffPartialType, SankoffPartialType::PLeft,
-                   SankoffPartialType::Q>;
-
   // DNA assumption
   static constexpr size_t state_count_ = 4;
   static constexpr double big_double_ = static_cast<double>(INT_MAX);
+  // this is for the pvhandler 
+  constexpr static double resizing_factor_ = 2.0;
```

Comment:
Perhaps this should stay in the PV handler? cc @davidrich27

Reply:
resolved